Lib/argparse.py

```diff
@@ -600,7 +600,11 @@ def _expand_help(self, action):
         if params.get('choices') is not None:
             choices_str = ', '.join([str(c) for c in params['choices']])
             params['choices'] = choices_str
-        return self._get_help_string(action) % params
+        expanded_help = self._get_help_string(action)
+        for param, value in params.items():
+            if f'%({param})s' in expanded_help:
```

Comment:
I think this conditional is redundant, right? Doesn't `replace()` handle validating that the string is contained by `expanded_help`?